src/test/java/com/jcabi/xml/StrictXMLTest.java

```diff
@@ -51,20 +51,27 @@
 public final class StrictXMLTest {
 
     /**
+     * Valid xml document.
+     */
+    private static final String VALID_XML = "<root>test</root>";
```

Comment:
by re-using these constants in multiple test methods you're telling the reader of your test case "these test methods have something in common". in reality, they have nothing in common. it's better to use different XML and XSD in each test method. see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#redundant-constants